src/validator.ts

```diff
@@ -66,13 +66,34 @@ export class Validator {
   // All defined parameters should have natspec
   private validateParameters(node: ErrorDefinition | FunctionDefinition | ModifierDefinition, natspecParams: (string | undefined)[]): string[] {
     let definedParameters = node.vParameters.vParameters.map((p) => p.name);
-    return definedParameters.filter((p) => !natspecParams.includes(p)).map((p) => `@param ${p} is missing`);
+    let alerts: string[] = [];
+    const counter = countElements(natspecParams);
```

Comment:
The name of the function sounds a bit like it just returns `array.length`. `countUniqueElemenents` or something similar might be more appropriate, wdyt?